streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractWindowBytesStoreTest.java

```diff
@@ -931,16 +979,6 @@ public void shouldThrowNullPointerExceptionOnGetNullKey() {
         assertThrows(NullPointerException.class, () -> windowStore.fetch(null, ofEpochMilli(1L), ofEpochMilli(2L)));
     }
 
-    @Test
-    public void shouldThrowNullPointerExceptionOnRangeNullFromKey() {
-        assertThrows(NullPointerException.class, () -> windowStore.fetch(null, 2, ofEpochMilli(1L), ofEpochMilli(2L)));
-    }
-
-    @Test
-    public void shouldThrowNullPointerExceptionOnRangeNullToKey() {
-        assertThrows(NullPointerException.class, () -> windowStore.fetch(1, null, ofEpochMilli(1L), ofEpochMilli(2L)));
-    }
-
```

Comment:
Instead of deleting maybe you want to change the name and check that the the store returns the right values.

Reply:
Yes, but I've already tested these 2 test cases above (i.e. `testFetchRange` and `testBackwardFetchRange`). I don't think we should test them again here. What do you think?

Reply:
Ok, no, if you have them covered up there that's fine.